### PR TITLE
Silence linkchecker's robots.txt warnings.

### PR DESCRIPTION
**Pull request by robinlinden**
This change is [Reviewable](https://reviewable.io/reviews/toktok/website/30)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -31,6 +31,9 @@ script:
 - curl https://git-critique.herokuapp.com/hello/changelog >> toktok/changelog/c-toxcore.md
 - curl https://git-critique.herokuapp.com/hello/roadmap >> toktok/roadmap/c-toxcore.md
 - make toktok-site
+- mkdir -p ~/.linkchecker/ && \
```
Comment (iphydf) [inline .travis.yml]:
```yaml
- >
    mkdir -p ... && \
    ...
```

or: `- mkdir -p ... && ...` without newline.

**Comment by iphydf** [comment]
:lgtm_strong:

---

Review status: 0 of 1 files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/website/30#-:-Ka8qBdk0qgtc58IpAmm:b-jbsf8z)*

**Comment by iphydf** [comment]
Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/website/30)*

**Comment by cebe** [comment]
:lgtm_strong:

---

Review status: all files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/website/30#-:-KaAz73yBJScLrZ3p25_:b-jbsf8z)*

**Comment by cebe** [comment]
:lgtm_cancel: see comment below.

---

Review status: all files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---

*[.travis.yml, line 38 at r2](https://reviewable.io:443/reviews/toktok/website/30#-KaFRmADa5xDTjttfvVH:-KaFRmADa5xDTjttfvVI:bruscxk) ([raw file](https://github.com/toktok/website/blob/11c04cc83fb728f4535d6dd5ef9a68e46354abe6/.travis.yml#L38)):*
> ```YAML
>     && echo "[filtering]" > ~/.linkchecker/linkcheckerrc \
>     && echo "ignorewarnings=http-robots-denied" >> ~/.linkchecker/linkcheckerrc
> - linkchecker --ignore-url "https://travis-ci.org.*" --ignore-url "irc://.*" toktok-site || true
> ```

this conflicts with the changes to makefile and travis.yml on master. This change should be moved to the makefile check target.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/website/30#-:-KaFRxakq0RXiLmDN5NT:bfc6qrl)*

**Comment by robinlinden** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

*[.travis.yml, line 38 at r2](https://reviewable.io:443/reviews/toktok/website/30#-KaFRmADa5xDTjttfvVH:-KaH_10ab1tVb9eREILG:bf8eefi) ([raw file](https://github.com/toktok/website/blob/11c04cc83fb728f4535d6dd5ef9a68e46354abe6/.travis.yml#L38)):*
<details><summary><i>Previously, cebe (Carsten Brandt) wrote…</i></summary><blockquote>

this conflicts with the changes to makefile and travis.yml on master. This change should be moved to the makefile check target.
</blockquote></details>

Done. Rebased on master and solved the merge conflict by moving this to the Makefile.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/website/30)*

**Comment by Zer0-One** [comment]
Reviewed 2 of 2 files at r3.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

*[Makefile, line 51 at r3](https://reviewable.io:443/reviews/toktok/website/30#-KaHbi7f02zPJAooD6zh:-KaHbi7f02zPJAooD6zi:bd5x8x3) ([raw file](https://github.com/toktok/website/blob/9b229790be0fb91c33513935a2ac985eaea9c146/Makefile#L51)):*
> ```Makefile
> 
> check:
> 	mkdir -p ~/.linkchecker/
> ```

Directory might already exist, add a "-" before "mkdir" to allow a failure. Also, if the user's home directory was missing, there's very little chance they'd actually have permission to create it; you can remove the "-p" option.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/website/30)*

**Comment by Zer0-One** [comment]
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

*[Makefile, line 51 at r3](https://reviewable.io:443/reviews/toktok/website/30#-KaHbi7f02zPJAooD6zh:-KaHe_uN56IZWBYaWStW:b32w0x) ([raw file](https://github.com/toktok/website/blob/9b229790be0fb91c33513935a2ac985eaea9c146/Makefile#L51)):*
<details><summary><i>Previously, Zer0-One (David Zero) wrote…</i></summary><blockquote>

Directory might already exist, add a "-" before "mkdir" to allow a failure. Also, if the user's home directory was missing, there's very little chance they'd actually have permission to create it; you can remove the "-p" option.
</blockquote></details>

OK

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/website/30)*

**Comment by cebe** [comment]
:lgtm_strong:

---

Reviewed 2 of 2 files at r3.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/website/30#-:-KaHz8Xx8L1YF26zDyHq:b-jbsf8z)*

**Comment by iphydf** [comment]
Reviewed 1 of 2 files at r3.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/website/30)*

**Comment by subliun** [comment]
:lgtm_strong:

---

Reviewed 2 of 2 files at r3.
Review status: :shipit: all files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/website/30#-:-KaIKSa4icf4KE3MalsB:b-jbsf8z)*

**Pull request merged**